<launch>
  <arg name="frame" default="crazyflie_test" />
  <arg name="world_frame" default="world"/>
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="file_dir" default="/home/samoa/catkin_ws/src/flow_field/data/" />

  <include file="$(find mocap_vicon)/launch/vicon.launch"/>

  <group ns="pitot">
    <node name="joy" pkg="joy" type="joy_node" output="screen">
      <param name="dev" value="$(arg joy_dev)" />
    </node>

    <node pkg="flow_field" type="data_recording.py" name="data_recording_node" output="screen">
      <param name="frame"  value="$(arg frame)"/>
      <param name="world_frame" value="$(arg world_frame)"/>
      <param name="file_dir" value="$(arg file_dir)"/>
    </node>

    <node pkg="flow_field" type="xbox_record_client.py" name="xbox_record_client_node" output="screen">
      <param name="use_crazyflie_controller" value="True" />
    </node>
    
  </group>


</launch>